backend/tests/accounts/test_views.py

```diff
@@ -413,6 +414,100 @@ def test_update_same_major_twice(self):
         self.assertEqual(response.status_code, 200)
 
 
+class ProfilePicViewTestCase(TestCase):
+    def setUp(self):
+        self.user = get_user_model().objects.create_user(
+            pennid=1,
+            username="student",
+            password="secret",
+            first_name="First",
+            last_name="Last",
+            email="test@test.com",
+        )
+
+        self.client = APIClient()
+        self.serializer = UserSerializer(self.user)
+
+    def test_profile_pic_upload_empty(self):
+        # empty image throws an error
+        resp = self.client.post(reverse("accounts:me-pfp-upload"))
+        self.assertIn(resp.status_code, [400, 403], resp.content)
```

Comment:
Is there a reason we assert the status code is either a `400` or `403`?

Reply:
No—that's just what the tests I copied from clubs did. I've replaced the `assertIn` tests with `assertEqual` tests